packages/react/src/components/header/Header.stories.tsx

```diff
@@ -26,7 +27,19 @@ export default {
   args: {},
 };
 
-const languageChangedAction = action('language:onChange');
+const languageChangedAction = () => {
+  action('language:onChange');
+};
+
+const logoSrcFromLanguage = (lang: string) => {
```

Comment:
I'll have a look on this. But yes the user has to change the src of the logo in the new way the Logo component works since the Logo is not an inline-svg anymore.

Reply:
And oh it might have been so that the header theme didn't exist when I made this :D

Reply:
Well fixed now for the stories that do work (the full-ones still crash for dark/light)